python_modules/libraries/dagster-airbyte/dagster_airbyte/resources.py

```diff
@@ -63,6 +69,33 @@ def api_base_url(self) -> str:
             + "/api/v1"
         )
 
+    @contextmanager
+    def cache_requests(self):
```

Comment:
what's the methodology for determining which requests to cache? Seems like this is currently only used when getting destination definition, but not when getting source definitions.

Reply:
Anything involving source/destination definitions should be very safe since they won't change during a reconciliation run, I've gone ahead and done a pass to update all of those.

We can probably expand this down the line, but some fields might change during reconciliation itself (e.g. sources/destinations getting updated).